test/refData/refBatchEmbedded.csv

```diff
@@ -1,11 +1,11 @@
 x1,x2
-1.4122797110018108,9.554717370430515
```

Comment:
Why has the ref data changed?

Reply:
The algorithm changed slightly in the update, there's some discussion about that in the paper https://f1000research.com/articles/8-2120 (mostly: I avoided some stuff that caused "jumps" in the data). If you look at the actual point values, they are usually "almost" in the same position as originally, +- some scaling and a bit or movement.